pages/about.html

```diff
@@ -1,12 +1,142 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="/styles/about.css">
+        <title>Resume</title>
+    </head>
+
+    <body>
+        <nav id="navbar">
+            <a id ="homeButton" href="index.html">Home</a>
+            <div id=navbar-right>
+                <a href="about.html">Resume</a>
+                <a href="portfolio.html">Projects</a>
+                <a href="index.html#contact">Contact</a>
+            </div>
+        </nav>
+
+        <div id="hjResume">
+            <section id="nameTitle">
+                <div id="nameLeft"></div>
+                <div id= "nameRight">
+                    <h1>Hyunjeong(Katherine) Otten Kim</h1>
+                </div>
+            </section>
+
+            <section class="content-box"> 
+                <div class="content-menu">
+                    <h3>Experience</h3>
+                </div>
+                <div class="content">
+                    <div class="exp">
+                        <h1 class="exp_title">Chemistry lab assistant</h1>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)